exporter/elasticsearchexporter/integrationtest/datareceiver.go

```diff
@@ -27,16 +27,18 @@ import (
 	"github.com/open-telemetry/opentelemetry-collector-contrib/testbed/testbed"
 )
 
+var emptyLogs plog.Logs
```

Comment:
nit: Can this be moved into `newMockESReceiver` to avoid a global var?

Reply:
Good catch! Fixed.